components/IcebreakerGenerator.tsx

```diff
@@ -17,7 +17,7 @@ export const IcebreakerGenerator = ({
 }: Props) => {
   return (
     <div className="flex w-full flex-col items-center justify-center divide-y">
-      <div className="flex items-center justify-center p-6 sm:p-6">
```

Comment:
@BartoszJarocki I'm new to Tailwind, and I'm a little confused about the `sm:` part of the padding classes. Is that supposed to mean that it only afftects the small sizes?

Reply:
@enriquesanchez tailwind encourages mobile-first design, so all the breakpoints specify `minimum width`. 
this means, for example, `p-2 sm:p-4 md:p-6` means small padding on the smallest devices, and the padding grows with the screen size. more on this here: https://tailwindcss.com/docs/responsive-design